plugins/cicd-statistics/src/plugin.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createPlugin,
+  createRoutableExtension,
+  createRouteRef,
+} from '@backstage/core-plugin-api';
+
+export const rootCatalogCicdStatsRouteRef = createRouteRef({
+  id: 'cicd-statistics',
+});
+
+export const cicdStatisticsPlugin = createPlugin({
+  id: 'cicd-statistics',
+  routes: {
+    entityContent: rootCatalogCicdStatsRouteRef,
+  },
+});
+
+export const EntityCicdStatisticsContent = cicdStatisticsPlugin.provide(
+  createRoutableExtension({
+    component: () => import('./entity-page').then(m => m.EntityPageCicdCharts),
+    mountPoint: rootCatalogCicdStatsRouteRef,
+    name: 'cicd-statistics-page',
```

Comment:
Should be `'EntityCicdStatisticsContent'`

Reply:
ok